### PR TITLE
🔗 Forge nemeton-1 chain

### PR DESCRIPTION
**Pull request by ad2ien**
Prepare `nemeton` chain for [incentivized program](https://nemeton.okp4.network/).

- [x] forge the genesis
- [x] set register dates
- [x] disable `update-peers` job (will be re-enabled when chain starts)
- [x] provide `README.md.gtpl` description
- [x] remove peers list for `nemeton`
- [x] set discontinued status in the readme for `nameton`

### CODE REVIEW

**Comment by ccamel** [comment]
@amimart I think we're done for this PR. I've deleted all account balances. I will introduce the different required accounts in another PR. It does not interfere with the registration of druites during the registration period.

File: chains/nemeton-1/README.md.gtpl [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+```sh
+# Init node
+okp4d --home mynode init your-moniker
+
+# Create keys, be careful with the mnemonic 👀
+okp4d --home mynode keys add your-key-name
+
+# Set account necessary balance
+okp4d --home mynode add-genesis-account your-key-name 1200000uknow
```
Comment (amimart) [inline chains/nemeton-1/README.md.gtpl:26]:
```suggestion
okp4d --home mynode add-genesis-account your-key-name 10000200000uknow
```

Regarding the first delegation of 10000 $KNOW they'll need a little more 😉

File: chains/nemeton-1/README.md.gtpl [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+```
+
+Then create your own genesis transaction (`gentx`). You will have to choose the following parameters for your validator: `commission-rate`, `commission-max-rate`, `commission-max-change-rate`, `min-self-delegation` (>=1), `website` (optional), `details` (optional), `identity` ([keybase](https://keybase.io) key hash, used to get validator logos in block explorers - optional), `security-contact` (email - optional).
+
+```sh
+# Create the gentx
+okp4d --home mynode gentx your-key-name 1000000uknow \
```
Comment (amimart) [inline chains/nemeton-1/README.md.gtpl:33]:
```suggestion
okp4d --home mynode gentx your-key-name 10000000000uknow \
```

I'm in favor to set the same amount as it is in the registration process.

**Review by amimart — CHANGES_REQUESTED** [review]
It seems almost good to me, just some remarks regarding the given example to create the gentx.

**Review by ccamel — APPROVED** [review]

**Review by bdeneux — APPROVED** [review]

**Review by amimart — APPROVED** [review]

**Pull request merged**